### ISSUE TITLE
menuItem Error 'Can't have both badge and subItems'

### ISSUE THREAD

**Issue by pssguy** [rstudio/shinydashboard#63]
Any particular reason for this?
Also could badgeLabel  and badgeColor be menuSubItem parameters, please
tx

**Comment by wch** [comment]
Yes, the reason is that there's no space for both the dropdown arrow and the badge.

I'll look into the possibility of adding the badge to menuSubItems.

**Comment by pssguy** [comment]
OK makes sense
Thanks for checking out use in menuSubItem. I use them quite a bit

**Comment by maxheld83** [comment]
@pssguy you might be interested to know that you can work around this by nesting `menuItem()`s only, as per [this hack](https://stackoverflow.com/questions/37597136/shinydashboard-is-it-not-possible-to-have-nested-menu-sub-items-cant-make-it). 
Looks the same, not sure whether there might be negative effects or limitations.

**Comment by NicolasImberty** [comment]
Hello

Is it possible add badge in  menuSubItem  ?
For example :
https://stackoverflow.com/questions/55039023/shinydashboard-badgelabel-in-menusubitem
